ftw/solr/search.py

```diff
@@ -33,10 +33,21 @@ def search(self, request_handler=u'/select', query=u'*:*', filters=None,
             filters = []
         if not isinstance(filters, list):
             filters = [filters]
-        filters.insert(0, self.security_filter())
+        if not unrestricted:
+            filters.insert(0, self.security_filter())
         params[u'filter'] = filters
         return conn.search(params, request_handler=request_handler)
 
+    def search(self, request_handler=u'/select', query=u'*:*', filters=None,
+               start=0, rows=1000, sort=None, **params):
+        return self._search(
+            request_handler, query, filters, start, rows, sort, False, **params)
```

Comment:
You are passing keyword arguments as positional arguments. This is discouraged as you will have trouble when the arguments are reordered in the method definition (As IMO it would be absolutely OK if somebody did so). Please change it to keyword arguments. This has the additional benefit that the currently naked `True` or `False` will be changed to `unrestricted=True` which improves readability by explaining the literal value.